vips/vip-300/bscmainnet.ts

```diff
@@ -0,0 +1,30 @@
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const NORMAL_TIMELOCK = "0x939bD8d64c0A9583A7Dcea9933f7b21697ab6396";
+
+const vip300 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-300 Ethereum: Add FRAX and sFrax markets to Core pool",
+    description: ``,
+    forDescription: "I agree that Venus Protocol should proceed with this proposal",
+    againstDescription: "I do not think that Venus Protocol should proceed with this proposal",
+    abstainDescription: "I am indifferent to whether Venus Protocol proceeds or not",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: NORMAL_TIMELOCK,
```

Comment:
```suggestion
        target: FAST_TRACK_TIMELOCK,
```

Reply:
Isn't it similar to https://app.venus.io/#/governance/proposal/295?chainId=56?

Reply:
In this case, we are adding new markets, not only changing risk parameters. Normal timelock is ok